src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesOperatorsAndCasts.java

```diff
@@ -108,8 +108,8 @@ private int f7() {
         return 0;
     }
 
-    static class TypeParameterized<T> {}
-    static class TypeA extends TypeParameterized<String> {}
+    static class GenT<T> {}
+    static class TypeA extends GenT<String> {}
```

Comment:
This is not required, please revert this.

Reply:
I will revert it. I made this change to make space [here](https://github.com/faraazb/checkstyle/blob/db6a1205a592e10bcd14b1845b531d11b857581f/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesOperatorsAndCasts.java#L95). I guess 'violation above' is better than wrapping in this case?

Reply:
Over there it is fine, here renaming to `GenT` is not of any use, there renaming made more space, I understand you are trying to be consistent with naming ut it doesn't matter in the test file unless it's a naming related check.

Reply:
Since this input file gets compiled I cannot rename (`TypeParameterized` to `GenT`) only on line 95. How should I go about?

Reply:
Oh ok, you are using this class as ref var type over there, it's fine.

Reply:
Resolved.